docker_templates/templates/snippet/vcs_import.Dockerfile.em

```diff
@@ -0,0 +1,9 @@
+@[if 'vcs' in locals()]@
+@[if vcs]@
+WORKDIR @(ws)
+@[for i, (imports_name, imports) in enumerate(vcs.items())]@
```

Comment:
Nitpick, I think it would be more flexible to specify the file name in the import name rather that hardcoding a `.repos` suffix to the name.
Alternatives are:
use `wget [...] -o @(imports_name).repos` or use the basename of the url passed to wget for the `vcs import` command